etc/kayobe/ansible/migrate-rabbitmq-queues.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Migrate RabbitMQ queues from HA to Quorum
+# This is primarily used in CI workflows
+
+- name: Migrate RabbitMQ queues
+  hosts: localhost
+  gather_facts: no
+  vars:
+    - rabbitmq_container_name: "rabbitmq"
+    - services_to_restart: "barbican,blazar,cinder,cloudkitty,designate,heat,ironic,keystone,magnum,manila,neutron,nova,octavia"
+  tasks:
+    - name: Checking timedatectl status
+      command: timedatectl status
+      register: timedatectl_status
+      changed_when: false
+
+    - name: Fail if the clock is not synchronized
+      assert:
+        that:
+          - "'synchronized: yes' not in timedatectl_status.stdout"
+        fail_msg: >
+          timedatectl sees the system clock as unsynchronized.
+          You may need to force synchronisation using `chronyc makestep`.
+          Otherwise, please wait for synchronization.
+
+    - name: Inspect the {{ rabbitmq_container_name }} container
+      shell:
+        cmd: "docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ rabbitmq_container_name }}"
+      register: inspection
+      become: true
+
+    - name: Ensure the {{ rabbitmq_container_name }} container is running
+      command: "systemctl start kolla-{{ rabbitmq_container_name }}-container.service"
+      when: inspection.stdout == 'false'
+      become: true
+
+    - name: Wait for the {{ rabbitmq_container_name }} container to reach state 'Running'
+      shell:
+        cmd: "docker container inspect --format '{{ '{{' }} .State.Running {{ '}}' }}' {{ rabbitmq_container_name }}"
+      register: result
+      until: result.stdout == 'true'
+      retries: 10
+      delay: 6
+      become: true
+
+    - name: Wait for the rabbitmq node to automatically start on container start
+      command: "docker exec {{ rabbitmq_container_name }} /bin/bash -c 'rabbitmqctl wait /var/lib/rabbitmq/mnesia/rabbitmq.pid --timeout 60'"
+      when: inspection.stdout == 'false'
+      become: true
+
+    - name: Migrate RabbitMQ queues
+      shell:
+        cmd: >
+          kayobe overcloud service configuration generate --node-config-dir /etc/kolla --kolla-skip-tags rabbitmq-ha-precheck -ke om_enable_rabbitmq_quorum_queues=true -ke om_enable_rabbitmq_high_availability=false &&
```

Comment:
If we intend for this playbook to be used outside of CI (we should, but it should go into Antelope), we'll want to make sure this gets persisted in globals.yml. Just in case someone runs the queue migration, but then doesn't immediately upgrade to Caracal.